tsnet/tsnet.go

```diff
@@ -146,6 +147,48 @@ func (s *Server) Start() error {
 	return s.initErr
 }
 
+// Up connects the server to the tailnet and waits until it is running.
+// On success it returns the current status, including a Tailscale IP address.
+func (s *Server) Up(ctx context.Context) (*ipnstate.Status, error) {
```

Comment:
probably should also call `Start`.

Reply:
added a comment that this is done by LocalClient